membrane/client/flask/__init__.py

```diff
@@ -219,17 +218,26 @@ def _redirect_for_authentication():
     return redirect(f"{_config.certificate.auth_url}/authenticate?token={jwt_token}")
 
 
+def _redirect_for_logout(user_id):
+    """"""
+    claims = {"sub": user_id}
+    jwt_token = _create_custom_token(request.url, custom_claims=claims)
+    return redirect(f"{_config.certificate.auth_url}/logout?token={jwt_token}")
+
+
 @blueprint.route("/login")
 def login():
     """Login route."""
-    if hasattr(membrane_current_user, "id"):
-        return redirect(_landing_page_url())
+    if membrane_current_user.is_authenticated:
+        return redirect(_landing_endpoint())
```

Comment:
an url would be where a browser gets redirected to

an endpoint would be part of an API called from a webapp running within the browser.

renaming from page_url to endpoint seems to be a worse choice.

Reply:
Done